src/main/java/com/amazonaws/encryptionsdk/AwsCrypto.java

```diff
@@ -87,6 +87,8 @@
 public class AwsCrypto {
     private static final Map<String, String> EMPTY_MAP = Collections.emptyMap();
 
+    private boolean pointCompressionEnabled_ = true;
```

Comment:
this keeps it backward compatible.

could you write a test case to ensure that the default value is always intact and is set to true?